testing/freeimage/APKBUILD

```diff
@@ -25,6 +25,9 @@ prepare() {
 }
 
 build() {
+	if [ "$CARCH" == "ppc64le" ]; then
+                unset CFLAGS CPPFLAGS CXXFLAGS
+        fi
```

Comment:
`checkbashisms` reports this:

````
possible bashism in APKBUILD line 28 (should be 'b = a'):
        if [ "$CARCH" == "ppc64le" ]; then
````

Reply:
I also wonder if we should unset the flags for all archs?

Reply:
I want avoid arch specific conditionals if possible.

What do you think about this patch instead:
````diff
diff --git a/Makefile.gnu b/Makefile.gnu
index 92f6358..293ee97 100644
--- a/Makefile.gnu
+++ b/Makefile.gnu
@@ -15,7 +15,8 @@ LIBRARIES = -lstdc++
 
 MODULES = $(SRCS:.c=.o)
 MODULES := $(MODULES:.cpp=.o)
-CFLAGS ?= -O3 -fPIC -fexceptions -fvisibility=hidden
+CFLAGS ?= -O3
+CFLAGS += -fPIC -fexceptions -fvisibility=hidden
 # OpenJPEG
 CFLAGS += -DOPJ_STATIC
 # LibRaw
@@ -23,7 +24,8 @@ CFLAGS += -DNO_LCMS
 # LibJXR
 CFLAGS += -DDISABLE_PERF_MEASUREMENT -D__ANSI__
 CFLAGS += $(INCLUDE)
-CXXFLAGS ?= -O3 -fPIC -fexceptions -fvisibility=hidden -Wno-ctor-dtor-privacy
+CXXFLAGS ?= -O3
+CXXFLAGS += -fPIC -fexceptions -fvisibility=hidden -Wno-ctor-dtor-privacy
 # LibJXR
 CXXFLAGS += -D__ANSI__
 CXXFLAGS += $(INCLUDE)
````

Reply:
Yes, that looks better and built fine on my ppc64le and x86_64 machines. I'll submit a new pull request. Thanks ncopa